File: buoi2/java/HelloWorld.java
/**
 * HelloWorld Java Application
 * Simple Hello World program in Java
 * Created for Network Programming course
 */
public class HelloWorld {
    
    /**
     * Main method - entry point of the application
     * @param args command line arguments
     */
    public static void main(String[] args) {
        // Display welcome messages
        System.out.println("Hello, World!");
        System.out.println("Chao mung den voi lap trinh Java!");
        System.out.println("========================================");
        
        // Display Java information
        System.out.println("Java Version: " + System.getProperty("java.version"));
        System.out.println("Java Vendor: " + System.getProperty("java.vendor"));
        System.out.println("OS Name: " + System.getProperty("os.name"));
        System.out.println("OS Version: " + System.getProperty("os.version"));
        System.out.println("User Name: " + System.getProperty("user.name"));
        System.out.println("========================================");
        
        // Simple interaction
        java.util.Scanner scanner = new java.util.Scanner(System.in);
        System.out.print("Nhap ten cua ban: ");
        String name = scanner.nextLine();
        
        System.out.println("Xin chao, " + name + "!");
        System.out.println("Chuc mung ban da chay thanh cong chuong trinh Java!");
        
        // Display some basic calculations
        System.out.println();
        System.out.println("--- Demo tinh toan don gian ---");
        int a = 10, b = 5;
        System.out.println(a + " + " + b + " = " + (a + b));
        System.out.println(a + " - " + b + " = " + (a - b));
        System.out.println(a + " * " + b + " = " + (a * b));
        System.out.println(a + " / " + b + " = " + (a / b));
        
        scanner.close();
    }
}

File: buoi2/HelloWorld.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-

"""
HelloWorld.py - Simple Hello World program in Python
Created for Network Programming course
"""

def main():
    """Main function to display Hello World message"""
    print("Hello, World!")
    print("Chào mừng đến với lập trình Python!")
    print("=" * 40)
    
    # Display some system information
    import sys
    import platform
    
    print(f"Python version: {sys.version}")
    print(f"Platform: {platform.platform()}")
    print(f"Architecture: {platform.architecture()}")
    print("=" * 40)
    
    # Simple interaction
    name = input("Nhập tên của bạn: ")
    print(f"Xin chào, {name}!")
    print(f"Chúc mừng bạn đã chạy thành công chương trình Python!")

if __name__ == "__main__":
    main()